PKGBUILD

```diff
@@ -1,56 +1,49 @@
 # Maintainer: Chris Salzberg <chris@dejimata.com>
-# Contributor: Leonidas Spyropoulos <artafinde@gmail.com>
 
-pkgname=neomutt
-pkgver=20170912
+_pkgname=neomutt
+pkgname=neomutt-git
+pkgver=neomutt.20171215.r0.gae612851
 pkgrel=1
 pkgdesc='The New Mutt: powerful text-based mail client with all the best feature patches'
 url='http://www.neomutt.org/'
 license=('GPL')
-validpgpkeys=('86C2397270DD7A561263CA4E5FAF0A6EE7371805') # Richard Russon (flatcap) <rich@flatcap.org>
-source=("https://github.com/neomutt/neomutt/archive/$pkgname-$pkgver.tar.gz"
-        "https://github.com/neomutt/neomutt/releases/download/$pkgname-$pkgver/$pkgname-$pkgver.tar.gz.sig")
-sha256sums=('9f0d128fe8dbbbddb67e2b629f8b5e52a9d892f82c735845c1708c14e3046657'
-            'SKIP')
+source=('git+https://github.com/neomutt/neomutt.git#branch=master')
+sha256sums=('SKIP')
 arch=('i686' 'x86_64')
-depends=('openssl' 'gdbm' 'mime-types' 'libsasl' 'gnupg' 'gpgme' 'libidn' 'krb5' 'notmuch-runtime')
+depends=('openssl' 'gdbm' 'mailcap' 'libsasl' 'gnupg' 'gpgme' 'libidn' 'krb5' 'notmuch-runtime' 'docbook-xsl')
 optdepends=('urlview: for url menu')
 makedepends=('git' 'gnupg' 'libxslt')
-conflicts=('mutt')
-provides=('mutt')
-
-prepare() {
-  cd $srcdir
-  mv "$pkgname-neomutt-$pkgver" "$pkgname-$pkgver"
-  cd "$pkgname-$pkgver"
-
-  ./prepare \
-    --prefix=/usr \
-    --sysconfdir=/etc \
-    --enable-debug \
-    --enable-pgp \
-    --enable-gpgme \
-    --enable-notmuch \
-    --with-gss=/usr \
-    --with-ssl=/usr \
-    --with-sasl \
-    --with-curses=/usr \
-    --with-regex \
-    --with-idn \
-    --with-gdbm
+conflicts=('neomutt')
+provides=('neomutt')
+
+pkgver() {
+  cd "${srcdir}/${_pkgname}"
+
+  # Get the version number.
+  git describe --long --tags | sed 's/^v//;s/\([^-]*-g\)/r\1/;s/-/./g'
 }
 
 build() {
-  cd "$pkgname-$pkgver"
+  cd "${srcdir}/${_pkgname}"
+
+  ./configure \
```

Comment:
Why move this out of `prepare`? It doesn't to actual building, but rather, just prepares the build.

Reply:
I based this on neomutt-git's PKGBUILD. You make a valid point and I will move it into prepare().